projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -330,6 +347,8 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
   _offset: number;
   _pageSize: number;
   _offsetEvent = -1;
+  private _draggedRow: any;
```

Comment:
Let's create a separate group for private member variables:

```suggestion

  private _draggedRow: any;
```